exchange-v2/test/v2/ExchangeV2.rarible.test.js

```diff
@@ -18,122 +18,122 @@ const { Order, Asset, sign } = require("../order");
 const EIP712 = require("../EIP712");
 const ZERO = "0x0000000000000000000000000000000000000000";
 const { expectThrow, verifyBalanceChange } = require("@daonomic/tests-common");
-const { ETH, ERC20, ERC721, ERC1155, ORDER_DATA_V1, TO_MAKER, TO_TAKER, PROTOCOL, ROYALTY, ORIGIN, PAYOUT, CRYPTO_PUNK, COLLECTION, enc, id } = require("../assets");
+const { ETH, ERC20, ERC721, ERC1155, ORDER_DATA_V1, ORDER_DATA_V2, TO_MAKER, TO_TAKER, PROTOCOL, ROYALTY, ORIGIN, PAYOUT, CRYPTO_PUNK, COLLECTION, enc, id } = require("../assets");
 
 contract("ExchangeV2, sellerFee + buyerFee =  6%,", accounts => {
-	let testing;
-	let transferProxy;
-	let erc20TransferProxy;
-	let transferManagerTest;
-	let t1;
-	let t2;
-	let libOrder;
-	let protocol = accounts[9];
-	let community = accounts[8];
-	const eth = "0x0000000000000000000000000000000000000000";
-	let erc721TokenId0 = 52;
+  let testing;
+  let transferProxy;
```

Comment:
too many changes in this file. could you revert it and change only needed content? there can be many conflicts in future

Reply:
fixed